sample/EasyTables/CSharp/QueueToInItemToOutItem/function.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "bindings": [
+    {
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "samples-workitems"
+    },
+    {
+      "type": "easyTable",
+      "name": "queriedItem",
+      "direction": "in",
+      "id": "{QueueTrigger}",
+      "tableName": "Item"
+    },
+    {
+      "type": "easyTable",
```

Comment:
Table name isn't actually required above. If it was using JObject in signature it would be. But here it's actually just ignored. I'll clean that up.

And yes this only works if you use POCO. So node and JObject signatures require table name.

---
In reply to: [56187812](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56187812) [](ancestors = 56187812)